src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,19 @@
+package duke.task;
+
+public class Event extends Task {
+    protected String time;
+
+    public Event(String description, String time) {
+        super(description);
+        this.time = time;
+    }
```

Comment:
Any reason why you chose to have 2 constructors (string and datetime) for Deadline but not Event?